app/src/main/java/com/pajato/android/gamechat/common/FragmentType.java

```diff
@@ -68,7 +70,7 @@
  */
 public enum FragmentType {
     chatEnvelope (ChatEnvelopeFragment.class, none, R.layout.chat_envelope),
-    chatGroupList (ChatShowGroupsFragment.class, chatMain, R.layout.chat_list),
+    chatGroupList (ChatShowGroupsFragment.class, chatGroup, R.layout.chat_list),
```

Comment:
This change kills the hamburger menu on the chat group fragment.  The 'chatMain' toolbar type is special.  I have no clue off the top of my head how to fix this but you'll figure something out. :-)